src/ui/digital-exchange/components/list/ComponentList.js

```diff
@@ -22,11 +26,66 @@ class ComponentList extends Component {
     );
   }
 
+  renderComponentGridView() {
+    const componentPairs = this.props.digitalExchangeComponents
```

Comment:
In grid view, do we have component pairs at every screen resolution?

Reply:
@gidesan that is not clear at this point, so I refactored it to fit any scenario, no pairs used any more.

Reply:
IMO it's far better now!